.changeset/gold-cows-wash.md

```diff
@@ -0,0 +1,6 @@
+---
+"@openproject/helm-charts": major
+---
+
+Unset the image tag so users have to specify it explicitly. The problem with using `14-stable` is that new releases will auto rollover
+and break as the seeder is not being run. Upgrades between versions should be explicit.
```

Comment:
I don't understand why it would break if seeders are not being run. I though seeders are not supposed to do any data migrations. Is it still right? Did you mean database migrations?

Reply:
That makes total sense now. Thanks for the clarification.